frontend/package.json

```diff
@@ -254,6 +254,11 @@
     ]
   },
   "jest": {
+    "transform": {
+      "\\.[jt]sx?$": "babel-jest",
+      "\\.mjs?$": "babel-jest"
```

Comment:
SmartyStreets added support for ES6 modules which is what the `.mjs` file extension refers to here. Although the SDK upgrade works fine in the app without any issues, Jest does not natively support ES6 modules yet. This config tells Jest that when we run our frontend tests via Jest, we want any of those files to be compiled using `babel-jest` so that it can process javascript code used in tests correctly. Otherwise, it runs into syntax errors because Jest is not expecting to parse ES6 module imports.